userguide/advanceduse/reverse-tethering.rst

```diff
@@ -1,34 +1,76 @@
 Reverse tethering
 =================
 
-This tutorial helps you to connect your Ubuntu Touch device to your computer to access the Internet on it.
+This tutorial helps you to connect your Ubuntu Touch device to your computer to access the internet on it.
 This is useful if there is no available Wi-Fi connection or you don't have a data subscription on your Ubuntu Touch device.
 
-Prerequisite: A Ubuntu Touch device with "Developer mode" on, attached via USB to the computer connected to the Internet.
+Prerequisite: A Ubuntu Touch device with "Developer mode" on, attached via USB to the computer connected to the internet.
+
+Tested: 2023-01-02 with Ubuntu Touch 16.04 (OTA-24) and Ubuntu 22.04.
 
 Steps
 -----
 
-1. On the Ubuntu Touch device: ``android-gadget-service enable rndis``
+1. On the Ubuntu Touch device:
+
+  - Run: ``android-gadget-service enable rndis``
+
+  - Reboot the phone.
+
+2. On your computer:
 
-2. On your computer: get your RNDIS IP address e.g:``hostname -I``
+  - Get your RNDIS IP address:
 
-3. On the Ubuntu Touch device: 
+    Run: ``ip a``
 
-      add a gateway: ``sudo route add default gw YOUR_COMPUTER_RNDIS_IP``
+    Look for an interface named ``usb0`` in the output.
 
-      add nameservers: ``echo "nameserver 8.8.8.8" | sudo tee /etc/resolv.conf``
+    For example::
 
-      refresh the DNS table: ``resolvconf -u``
+        5: usb0: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc fq_codel state UNKNOWN group default qlen 1000
+           link/ether 76:eb:81:18:e0:40 brd ff:ff:ff:ff:ff:ff
```

Comment:
it can be argued that a mac address is personal, consider replacing (parts of) it with xx or so

Reply:
Good point. Didn'nt think about that. I'll change to `12:34:56:ab:cd:ef`. However in case of RNDIS these MAC addresses are randomly generated each to time you connect the phone. So in this particular case it's no personal information.

Reply:
lol. didn't think of that